src/pages/index.js

```diff
@@ -95,13 +160,72 @@ openProfileModalButton.addEventListener("click", () => {
 });
 
 
-//userInfo
-const profileName = ".profile__title";
-const profileDescription = ".profile__description";
-// Create user info instance
-const userInfo = new UserInfo({ userName: profileName, userJob: profileDescription});
 
 
+function handleCardDelete(card) {
+  confirmPopup.open();
+  confirmPopup.setSubmit(() => {
+    api.deleteCard(card._id).then(() => {
+      card.remove();
+      confirmPopup.close();
+    });
+  });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**